instructions/Installation.md

```diff
@@ -42,16 +42,21 @@ At the end execute it:
 ```shell
 ./mac_czkawka_gui
 ```
+#### Compiling on Apple Silicon
+Prebuilt binaries are available only for x86_64, so if you use ARM e.g. Mac M1/M2, you must compile the app manually.  
 
-**Warning**  
-Prebuilt binaries are available only for x86_64, so if you use ARM machine like e.g. Mac M1, you need to compile manually app.  
+**Compilation is simple, and takes less than 10 minutes on an M1/M2 mac.**
 
+User [@bauchdj](https://github.com/bauchdj) wrote [working rust complication instuctions](https://github.com/qarmin/czkawka/issues/689#issuecomment-1310549801)
```

Comment:
Steps should be pasted directly into this place to be able to update this steps if needed.